Optimize replication-between-clusters (#1390)
vesoft-inc/nebula-docs: docs-2.0/synchronization-and-migration/replication-between-clusters.md

> ### How to deal with synchronization failures?

Reviewer: Can we change this title to like【What should we do once synchronization problems/issues are fixed?】
The current title lets me think of solutions to the problem instead of what we should do after the problem is fixed.

Author: Thx but that's not the question.

> Fix the problems in the cluster, and the synchronization will be automatically restored.

Reviewer: ```suggestion
Fix the problems in the cluster, and then the synchronization will be automatically restored.
```

Reviewer: will it be better to add a conjunction to show the logic of this sentence?

> - If problems happened on the primary cluster, the synchronization would continue when the problems were fixed and the primary cluster restarted.

Reviewer: ```suggestion
- If problems have happened on the primary cluster, the synchronization continues when the problems are fixed and the primary cluster restarts.
```

Author: Great! Thanks for reminding me about how to write this kind of content!

> - If problems happened on the secondary cluster, listeners, or drainers, when the problems were fixed, the services that had the problems would receive the WALs accumulated from its upstream and the synchronization would continue. If the faulty machine were replaced with a new one, all the data of the synchronization services on the faulty machine must be copied to the new machine. Otherwise, the synchronization of the complete data set would start automatically.

Reviewer: ```suggestion
- If problems have happened in the secondary cluster, listeners, or drainers, when the problems are fixed, the services that had the problems will receive the WALs accumulated from its upstream and the synchronization will continue working. If the faulty machine is replaced with a new one, all the data of the synchronization services on the faulty machine must be copied to the new machine. Otherwise, the synchronization of the complete data set will start automatically.
```
